src/main.ts

```diff
@@ -1976,9 +1976,7 @@ function generateToJson(
     const readSnippet = (from: string): Code => {
       if (isEnum(field)) {
         const toJson = getEnumMethod(ctx, field.typeName, "ToJSON");
-        return isWithinOneOf(field)
```

Comment:
This clean up is great! Unfortunately it is rare when this `readSnippet`/etc code gets simpler, so really appreciate it.